packages/remix-dev/compiler/plugins/serverBareModulesPlugin.ts

```diff
@@ -82,6 +82,17 @@ export function serverBareModulesPlugin(
             return undefined;
         }
 
+        for (let transpileMatcher of remixConfig.serverDependenciesToBundle) {
```

Comment:
```suggestion
        for (let matcher of remixConfig.serverDependenciesToBundle) {
```

I'm on mobile otherwise I would rename all the variables below as well. This is more of a nitpick so feel free to ignore.